### ISSUE TITLE
Me might consider an alternative to django-endless-pagination (or fork it)

### ISSUE THREAD

**Issue by adrienbrunet** [nlhkabu/connect#26]
Last commit 2years ago. 13pull requests pending. :bomb: 

Maybe javascript could do the job. Any suggestions?

**Comment by Natim** [comment]
@rodo will tell you that we don't need pagination, we can display the 20 first profile and let the user play with filter to see the 20 mentors he is looking for.

**Comment by rodo** [comment]
I agree,  if the 20 first mentors not match you criterias may be more precise criterias is the solution

**Comment by Natim** [comment]
It will be even more accurate when we wil be able to zoom on the map to refresh the page as airbnb.com does.

**Comment by adrienbrunet** [comment]
:+1:

**Comment by nlhkabu** [comment]
piece of :cake:

**Comment by nlhkabu** [comment]
Or rather, :v: of :cake:

**Comment by nlhkabu** [comment]
In the short term, I suggest we remove endless pagination and we can go from there :)

**Comment by nlhkabu** [comment]
I have now removed endless pagination, but have not yet limited the queryset to only return 10-20 users.  Whilst I agree that 20 results should be enough when the searcher uses an accurate filter, I don't see why we have to limit this number - if someone wants to browse the results of a broad search, why stop them?  I do, however, think that pagination should work automatically, with JS loading new results as you approach the bottom of the page.
